engine/source/materials/fail/orthstrain/fail_orthstrain_s.F

```diff
@@ -750,13 +750,12 @@ SUBROUTINE FAIL_ORTHSTRAIN (
             NINDX   = NINDX + 1  
             INDX(NINDX) = I
             IF (UELR(I) >= NPT) THEN
-		            UVAR(I,31) = ONE
-              TDEL(I) = TIME 
-              OFF(I)  = FOUR_OVER_5 
-              P = THIRD*(SIGNXX(I) + SIGNYY(I) + SIGNZZ(I))
-              SIGNXX(I) = P
-              SIGNYY(I) = P
-              SIGNZZ(I) = P
+		          UVAR(I,31) = ONE
```

Comment:
could you correct , there are tabulation in this line

Reply:
Hi Sebastien, 

I have removed the tabulation in this line. 

Thanks a lot.